.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+// For format details, see https://aka.ms/vscode-remote/devcontainer.json or this file's README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.117.1/containers/python-3-miniconda
+{
+	"name": "PyMC4 Development Container",
+	"context": "..",
+	"image": "registry.hub.docker.com/ericmjl/pymc4:devcontainer",
```

Comment:
Btw, someone owns the pymc Docker repository namespace, so I don't have privileges to set this up. @twiecki or @fonnesbeck, do any of you know who is in charge of it?

Reply:
I've got the necessary permissions.